server/chat-commands/info.ts

```diff
@@ -2465,6 +2465,7 @@ export const commands: ChatCommands = {
 
 	'!pi': true,
 	pi(target, room, user) {
+		if (!this.runBroadcast()) return false;
```

Comment:
I'm honestly not sure, should commands return `false` or not? This is a question to @Zarel.

Reply:
It's complicated. I don't mind them returning `false`.